runtime/compiler.Dockerfile

```diff
@@ -52,3 +52,33 @@ RUN  set -xe \
  && ./bootstrap --prefix=/usr/local \
  && make \
  && make install
+
+# BISON parser - 3.0 - required by PHP
```

Comment:
Bison and re2c are only required when building from git, not when using release tarballs. Assuming this is not building from git, these *shouldn't* be necessary (though maybe some checks are broken?)

Reply:
It's built from `https://github.com/php/php-src/archive/php-7.4.0beta2.tar.gz`. And these checks failed, otherwise I wouldn't be adding bison and re2c.

It's also weird to me that previous versions were able to be built without `sqlite-devel` package installed but 7.4 requires it.

Feel free to try it - clone my branch with these changes and run `make build` inside the `runtime/` directory. Remove these lines installing bison and re2c beforehand and you should get to the same error about these dependencies...

Reply:
@nikic we are downloading the PHP source from https://github.com/php/php-src/archive/php-${VERSION_PHP}.tar.gz

It is an archive URL (not the git repository), but is that what you meant by "release tarballs"? The official link in the PHP blog is https://downloads.php.net/~derick/php-${VERSION_PHP}.tar.gz which looks weird, but why not (it's a RC after all).

I gave it a try and now I get an error about oniguruma not being found.

FYI this didn't solve the `oniguruma` problem (currently in the PR oniguruma is compiled from source, but still not detected by PHP):

```diff
- RUN LD_LIBRARY_PATH= yum install -y readline-devel gettext-devel libicu-devel libpng-devel libjpeg-devel sqlite-devel
+ RUN LD_LIBRARY_PATH= yum install -y readline-devel gettext-devel libicu-devel libpng-devel libjpeg-devel sqlite-devel oniguruma oniguruma-devel
```

Reply:
OK I'm making some progress! PHP is compiling, I am just missing `pecl` now, but it's probably not much.

The fix: eddc5deb37f6a7e734c0b253409bf0f7bb898ee7 (took me a while 😅)